content/en/blog/2020-10-30-charts-repo-deprecation.md

```diff
@@ -0,0 +1,38 @@
+
+When we announced the deprecation we also began to share the tools and guidance that we had used to maintain the helm/charts repo. For folks that want to host and maintain their own repositories you now have these tools available to streamline the process:
+
+- [Chart Testing](https://github.com/helm/chart-testing) provides linting and testing for PRs against your charts
+- [Chart Releaser](https://github.com/helm/chart-releaser) provides tooling to help you host your own chart repo with GitHub Relases and Pages used to host your artifacts
+- [Testing and Releasing Github Actions](https://github.com/helm?q=chart+action) to automate the tooling described above using GitHub Actions
+
+With these tools available we've enabled [many charts](https://github.com/helm/charts/issues/21103) to migrate to their own repositories for active maintainence.
+
+## Key Dates and Recommended Actions
+
+There has been refinement to the plans and confusion/questions about what happens next, so we wanted to provide a timeline of key events and **recommended actions** moving forward:
+
+* Nov 2, 2020 - READMEs for all non-deprecated charts will get a note added, stating that they will no longer be updated
+    * **RECOMMENDED ACTION** - If you depend on a chart in the Charts repository look for the new maintainance location. If one does not exist consider adopting the chart.
+* Nov 6, 2020 the stable and incubator charts repos will be removed from the [Artifact Hub](https://artifacthub.io/)
+    * **RECOMMENDED ACTION** - None
+* Nov 13, 2020 - CI on the [helm/charts repository](https://github.com/helm/chart) will be disabled and no more Pull Requests will be accpeted.
+    * **RECOMMENDED ACTION** - For more info on the ongoing initiative to relocate charts to new repos please see [this issue].(https://github.com/helm/charts/issues/21103)
+* *After* Nov 13, 2020 - Downloads of Charts at their old locations will be re-directed to the read-only archive avaliable in GH pages. The old locations may no longer be available after this date.
+    * **RECOMMENDED ACTION** - Info on switching to the archive are [here](https://helm.sh/docs/faq/#i-am-getting-a-warning-about-unable-to-get-an-update-from-the-stable-chart-repository). Keep in mind that these charts will no longer be updated with bug fixes or security patches.
```

Comment:
Using words like "here" as a target for links isn't ideal - can we rephrase like:

> Info on switching to the archive are [here](https://helm.sh/docs/faq/#i-am-getting-a-warning-about-unable-to-get-an-update-from-the-stable-chart-repository)

See info on [switching to use the archived `stable` and `incubator` charts](https://helm.sh/docs/faq/#i-am-getting-a-warning-about-unable-to-get-an-update-from-the-stable-chart-repository)